### DISCUSSION TITLE
Bahçe

### DISCUSSION THREAD

**Discussion by Deniz34000** [sonsuzus/sonsuzus#507] [Soru Cevap]
Luba bahçeyi olabildiğince hızlı sulamak için kovalardan birini seçmelidir ( yukarıda belirtildiği gibi, her saat i -inci kovayı seçerse a i uzunluğunda sürekli bir alt bölümü sulayacaktır ). Bahçeyi sulamak için harcaması gereken minimum saat sayısını belirlemesine yardımcı olun. Luba'nın her zaman bir kova seçebileceği ve böylece bahçeyi sulayabileceği garantilidir.

**Comment by Deniz34000** [comment]
def min_sulama_zamani(n, k, kovalar):
    min_saat = float('inf') 
    for a in kovalar:
        if k % a == 0:  
            saat = k // a  
            min_saat = min(min_saat, saat)  
    return min_saat
n, k = map(int, input().split())
kovalar = list(map(int, input().split())) 
print(min_sulama_zamani(n, k, kovalar))``

**Reply by sonsuzus** [reply]
Kod içine almalısın kodunu. Ayrıca sorunun açıklaması yetersiz.
